utility_manager.py

```diff
@@ -0,0 +1,147 @@
+#import helpers
+import importlib
+import sys
+import os
+
+#kivy helpers
+from kivy.uix.button import Button
+from kivy.uix.textinput import TextInput
+from kivy.uix.popup import Popup
+from kivy.uix.filechooser import FileChooserListView
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.label import Label
+from kivy.uix.scrollview import ScrollView
+from kivy.uix.widget import Widget
+
+#Helper class for the PluginPlayer application to browse files, view class types, and import new class types.
+class UtilityManager():
+    def __init__(self, plugin_player):
+        self.plugin_player = plugin_player
+    
+    
+    #browse for a new file from file system and place in entry
+    def browse(self):
+        popup = None
+
+        #grab text from entry
+        entry_text = self.plugin_player.root.ids.file_entry.ids.file_path_input.text
```

Comment:
```suggestion
         #grab text from entry
        entry_text = self.plugin_player.root.ids.file_entry.ids.file_path_input.text
```
You don't need to declare the variable ahead of time.